package.json

```
@@ -13,7 +13,7 @@
     "build": "turbo run build",
     "tsc": "tsc",
     "export-dynamic": "turbo run export-dynamic --concurrency 1",
-    "clean-dynamic-sources": "turbo run clean-dynamic-sources",
+    "copy-dynamic-plugins": "yarn --cwd dynamic-plugins/_utils run copy-dynamic-plugins",
```

should this be `copy-plugins` to match what you have in https://github.com/janus-idp/backstage-showcase/pull/631/commits/abae69b3b1bce3ea05c00ff60b6e40c359735280#diff-1e81b0dafa0098fa49ca9816f8ef7ee6bea94c697247a535821f294d85af2798R8 ?

```
Usage: node copy-plugins.js <destination directory>
```

Also I don't see the code that defaults to current dir if no parameter passed to `copy-plugins.js` but I guess it's in there somewhere...

```suggestion
    "copy-dynamic-plugins": "yarn --cwd dynamic-plugins/_utils run copy-plugins",
```

---

No, it isn't. The argument is required.

---

> should this be copy-plugins to match what you have in

see my answer here: https://github.com/janus-idp/backstage-showcase/pull/631#discussion_r1367363970

But I could rename the Javascript file if you think it is better, thoguh it would make it a bit redundant.
OTOH it seems to me that it is quite usual to have various commands called from `package.json` script entries, not necessarily having the same name as the command  (sometimes even there are several commands called from a single script entry).